Gemfile

```diff
@@ -6,6 +6,7 @@ gem 'faraday', '~> 1.0'
 gem 'faraday_middleware', '~> 1.0'
 
 gem 'jets'
+gem 'jwt'
```

Comment:
It's a best practice to specify atleast the major version in the Gemfile using the `~>` syntax.

Reply:
Thanks. I've made the changes.